patches/chrome-installer-mac-signing-signing.py.patch

```diff
@@ -1,49 +1,32 @@
-index ddd5aeef9dafb3e80ad20a7481c4f8ebb418493d..2fda1db466d960856ad397fb8fe97992871fbcec 100644
+index de69f7ba6cca5729ab83ff076211be36354d0f60..74a58cd639380039a5e2fd32ebb3384959eb6f46 100644
 --- a/chrome/installer/mac/signing/signing.py
 +++ b/chrome/installer/mac/signing/signing.py
-@@ -42,7 +42,6 @@ def get_parts(config):
-                 options=CodeSignOptions.RESTRICT,
-                 requirements=config.codesign_requirements_outer_app,
-                 identifier_requirement=False,
--                resource_rules='app_resource_rules.plist',
-                 entitlements='app-entitlements.plist',
-                 verify_options=VerifyOptions.DEEP + VerifyOptions.NO_STRICT),
-         'framework':
-@@ -82,9 +81,17 @@ def get_parts(config):
-                 options=CodeSignOptions.RESTRICT +
-                 CodeSignOptions.LIBRARY_VALIDATION,
-                 verify_options=VerifyOptions.IGNORE_RESOURCES),
-+        'sparkle-framework':
-+            CodeSignedProduct(
-+                '{.framework_dir}/Frameworks/Sparkle.framework'
-+                .format(config),
-+                'org.sparkle-project.Sparkle',
-+                verify_options=VerifyOptions.DEEP + VerifyOptions.NO_STRICT),
-     }
+@@ -124,6 +124,8 @@ def get_parts(config):
+             library_basename.replace('.dylib', ''),
+             verify_options=VerifyOptions.DEEP)
  
-     dylibs = (
-+        'libchallenge_bypass_ristretto.dylib',
-+        'libadblock.dylib',
-         'libEGL.dylib',
-         'libGLESv2.dylib',
-         'libswiftshader_libEGL.dylib',
-@@ -141,7 +148,7 @@ def sign_part(paths, config, part):
++    from signing_helper import AddBravePartsForSigning
++    AddBravePartsForSigning(parts, config)
+     return parts
+ 
+ 
+@@ -168,7 +170,7 @@ def sign_part(paths, config, part):
          part: The |model.CodeSignedProduct| to sign. The product's |path| must
              be in |paths.work|.
      """
 -    command = ['codesign', '--sign', config.identity]
 +    command = ['codesign', '--force', '--sign', config.identity]
-     if part.sign_with_identifier:
-         command.extend(['--identifier', part.identifier])
-     reqs = part.requirements_string(config)
-@@ -231,6 +238,9 @@ def sign_chrome(paths, config):
+     if config.notary_user:
+         # Assume if the config has notary authentication information that the
+         # products will be notarized, which requires a secure timestamp.
+@@ -262,6 +264,9 @@ def sign_chrome(paths, config, sign_framework=False):
  
-     # Sign the framework bundle.
-     sign_part(paths, config, parts['framework'])
-+    from signing_helper import GenerateWidevineSigFile
-+    GenerateWidevineSigFile(paths, config, parts['framework'])
-+    sign_part(paths, config, parts['framework'])
+         # Sign the framework bundle.
+         sign_part(paths, config, parts['framework'])
++        from signing_helper import GenerateWidevineSigFile
```

Comment:
This line can be deleted by combining with https://github.com/brave/brave-core/pull/2976/files#diff-bbb0120a0f4be9c6db810bfca1367681R9.
Also, can you help making below two lines into one line by calling `sign_part()` from `GenerateWidevineSigFile`.
I have a pending PR for it and something you did in here - https://github.com/brave/brave-core/pull/2789

Reply:
@simonhong, updated, please let me know if that's what you meant.